### DISCUSSION TITLE
System requirement for running OME

### DISCUSSION THREAD

**Discussion by deekshithkv** [OvenMediaLabs/OvenMediaEngine#1120] [Help]
What are the minimum system requirements for running OME without any issue to stream LLHLS from RTSP pull? Is there any cross compilation doc for different architecture available? Please share your comments.

**Comment by getroot** [comment]
There are too many variables depending on video quality, number of input streams, number of viewers, etc. Test it yourself using the Performance tool. Depending on conditions, it can be serviced on 1Core / 16M server.

https://airensoft.gitbook.io/ovenmediaengine/performance-tuning

**Comment by deekshithkv** [comment]
![image](https://user-images.githubusercontent.com/48976953/222630430-f01d0d67-3e18-4722-905c-a4f7d9dc8a70.png)

After running OME and started streaming ll-hls, i see a increase in buffer/cache size and it is not reduced even when OME service is stopped, is it expected? Is there a way to clear the buffer after the segments are streamed?
Where does OME save the playlist and segments for ll-hls? Is it in In-memory?

**Comment by getroot** [comment]
You may find it helpful to read the other issues below.

https://github.com/AirenSoft/OvenMediaEngine/issues/951

What does "OME Service is stopped" mean? Did you kill the process?

**Comment by deekshithkv** [comment]
yes killed the process, I am testing OME in ubuntu 20, Have used systemctl command to start and stop OME service

**Comment by getroot** [comment]
There will never be a case where any process does not return memory after it exits. If that happens, it's a kernel problem, but I don't think it is. It would be good to check which process is holding memory with ps or top command.
